config/owasp/suppressions.xml

```diff
@@ -7,6 +7,13 @@
     </notes>
     <cve>CVE-2016-1000027</cve>
   </suppress>
+  <suppress>
+    <notes>False Positives
```

Comment:
Could we add a date to this one like we do here? https://github.com/hmcts/darts-gateway/blob/dd4a252245e408ebd2880f00941e54628c0e7bd6/config/owasp/suppressions.xml#L13 
That way we won't suppress it forever, if there is a fix that gets released.